src/content-handlers/iiif/modules/uv-openseadragoncenterpanel-module/OpenSeadragonCenterPanel.ts

```diff
@@ -198,12 +198,28 @@ export class OpenSeadragonCenterPanel extends CenterPanel<
 
   updateResponsiveView(): void {
     this.setNavigatorVisible();
+    this.viewer.autoHideControls = true;
```

Comment:
It looks like this change has removed the original logic to disable auto-hide on mobile devices. Perhaps you should initialize this like:

```
this.viewer.autoHideControls = this.extension.isDesktopMetric();
```